scripts/lib/mlkem_test.py

```diff
@@ -307,32 +319,27 @@ def expect(scheme: SCHEME, actual: str) -> tuple[bool, str]:
                 f"Failed, expecting {expect}, but getting {actual}" if fail else "",
             )
 
-        return self._func.run_schemes(
+        return self._f(TEST_TYPES.MLKEM).test(
```

Comment:
From the commit message I was expecting to see pairs of compile+run being replaced by test, but here we _only_ call `run_schemes` before, while the new call to `test()` would additionally compile. Is that deliberate?

Reply:
originally, the paris of compile+run is in the local function `_func` in `func`
so the `self._func` is actually a replacement of the local function `_func` in `self.func`

so before the function calls was like
```
func -> local _func -> self._func.compile+ self._run_func -> self._func.run_schemes
all -> self._func.compile + self._run_func -> self._func.run_schemes
```
which is pretty hard to understand

it now became
```
func -> self._func -> self.__func.test -> self.__func.compile + self.__func.run_schemes
all -> self._func -> self.__func.test -> self.__func.compile + self.__func.run_schemes
```